test/e2e/framework.go

```diff
@@ -384,14 +387,18 @@ func (data *TestData) deleteAntrea(timeout time.Duration) error {
 // createPodOnNode creates a pod in the test namespace with a container whose type is decided by imageName.
 // Pod will be scheduled on the specified Node (if nodeName is not empty).
 func (data *TestData) createPodOnNode(name string, nodeName string, imageName string) error {
-	sleepDuration := 3600 // seconds
+	var commands []string
+	if imageName == "busybox" {
+		commands = []string{"sleep", strconv.Itoa(3600)} // 3600 seconds
+	}
```

Comment:
This is somewhat hacky, could command be an argument?

Reply:
done